test.js

```diff
@@ -133,6 +138,46 @@ describe('v8flags', function () {
     v8flags(function (err, flags) {
       expect(err).to.be.null;
       done();
-    })
+    });
+  });
+});
+
+describe('config-path', function () {
+  const moduleName = 'js-v8flags';
+
+  beforeEach(function() {
+    env.HOME = 'somehome';
+    cleanup();
+  });
+  afterEach(cleanup);
+
+  it('should return default linux path in other environments', function(done) {
+    delete require.cache[require.resolve('./config-path.js')];
```

Comment:
It’d be best to make it so it isn’t, then. Is this somehow clearing the cache for `user-home`?